cockroachdb/values.yaml

```diff
@@ -221,10 +220,15 @@ service:
         port: 26257
         # If using Istio set it to `cockroach`.
         name: grpc-internal
+
     http:
       port: 8080
       name: http
 
+    metrics:
```

Comment:
Can we just use the http port instead of introducing new variables? I'm not sure why you'd ever need to use a different target port for the metrics service than the port each pod is configured to serve its HTTP metrics on.